chapter_23_debugging_prod.asciidoc

```diff
@@ -328,8 +328,28 @@ Here's where we can put an early return in the FT:
     email = mail.outbox.pop()
 ----
 ====
-// CSANAD: TODO check if this would NOT return if we run the test against the
-// staging server, then?
+// CSANAD: This results in sending a real email to `edith@example.com` by
```

Comment:
@hjwp I received mails like this when I ran `test_login`. Can you check if you did too? I think you should have, because moving the early return higher up resolved it (clearly, since I'm no longer submitting the entered email address which caused the email to be sent) but I just want to cross-check this.

![mailer_daemon](https://github.com/user-attachments/assets/57d850d9-9db4-4b9b-a613-49a4d04afd2b)